hadoop-hdds/annotations/src/main/java/org/apache/ozone/annotations/RequestFeatureValidatorProcessor.java

```diff
@@ -1,13 +1,13 @@
 /*
  * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with this
- * work for additional information regarding copyright ownership.  The ASF
```

Comment:
I think we should keep double space between sentences to minimize future changes: almost all files have double space.

```
$ rg -Fl 'agreements. See' | wc -l                                                                                                                                        
24

$ rg -Fl 'agreements.  See' | wc -l
3886
```

Similarly:

```
$ rg -Fl 'ownership. The' | wc -l
24

$ rg -Fl 'ownership.  The' | wc -l
3485
```